### PR TITLE
origin_info mapping specs and normalization - a few tweaks

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made?

- beef up mapping specs for originInfo in diff flavors of empty
- mods normalization removes empty originInfo elements
- originInfo mapping specs about event type presentation collated and deduped ... and xit'ed the inconsistent one.

## How was this change tested?

it's all specs except mods normalization, which had a spec written for it.

### CODE REVIEW

File: spec/services/cocina/mapping/descriptive/mods/origin_info_date_spec.rb [hunk 1]
```diff
@@ -1335,29 +1335,162 @@
...
+          <originInfo eventType="presentation">
+            <dateCreated>1990</dateCreated>
+          </originInfo>
+        XML
+      end
+    end
+
+    context 'with more complex presentation' do
+      # from druid:ht706sj6651
+
```
Comment (ndushay) [inline spec/services/cocina/mapping/descriptive/mods/origin_info_date_spec.rb:1367]: moved all the presentation event type specs together and found a dup and removed it.

**Review by mjgiarlo — APPROVED** [review]
LGTM. since this includes a normalization change, should we test on sdr-deploy before shipping?

**Comment by ndushay** [comment]
### Main branch:

```
Status (n=5000):
  Success:   4898 (97.96%)
  Different: 61 (1.22%)
  To Cocina error:     0 (0.0%)
  To Fedora error:     0 (0.0%)
  Missing:     41 (0.82%)
```

# this branch
```
Status (n=5000):
  Success:   4898 (97.96%)
  Different: 61 (1.22%)
  To Cocina error:     0 (0.0%)
  To Fedora error:     0 (0.0%)
  Missing:     41 (0.82%)

```

**Comment by ndushay** [comment]
Merging myself because I even did a bigger roundtrip sample than above and the numbers were the same.

**Pull request merged**